aggsender/validator/proto/v1/validator.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option go_package = "github.com/agglayer/aggkit/aggsender/validator/proto/v1";
+package aggkit.aggsender.validator.v1;
+
+import "agglayer/node/types/v1/certificate.proto";
+import "agglayer/node/types/v1/certificate_id.proto";
+import "google/protobuf/empty.proto";
+
+// Service for validating new certificates
+service AggsenderValidator {
+    // Method to validate a new certificate
+    rpc ValidateCertificate(ValidateCertificateRequest) returns (google.protobuf.Empty);
+}
+
+// Request to validate a certificate
+message ValidateCertificateRequest {
+  // Previous certificate identifier.
```

Comment:
Nitpick: remove the dot because of consistency with the other comments
```suggestion
  // Previous certificate identifier
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/711/commits/8426eb824e2e249d622e420bc3f19ae221fbf9a0